types/lodash/common/collection.d.ts

```diff
@@ -1781,6 +1781,10 @@ declare module "../index" {
             collection: List<T> | null | undefined,
             callback: ValueIteratee<T>
         ): [T[], T[]];
+        partition<T1 | T2>(
```

Comment:
I don't think this is valid syntax? Should be `<T1, T2>`.

Reply:
Right. Sorry I made this change blindly

Reply:
Fixed